examples_tests/39.DenoiserTonemapper/main.cpp

```diff
@@ -423,10 +424,9 @@ void main()
 
 		return imageIDString;
 	};
-	bool* tileImageCheck = new bool[inputFilesAmount];
 	core::vector<ImageToDenoise> images(inputFilesAmount);
 	// load images
-	uint32_t maxResolution[EII_COUNT][2] = { 0 };
+	uint32_t maxResolution[2] = { 0,0 };
```

Comment:
no still keep separate maxResolution per denoiser type! just keep this as it was!